tmpl/shell.nix.tmpl

```diff
@@ -27,6 +27,15 @@ mkShell {
       export IN_NIX_SHELL=0
       export DEVBOX_SHELL_ENABLED=1
 
+      # Undo the effects of `nix-shell --pure` on SSL certs.
+      # See https://github.com/NixOS/nixpkgs/blob/dae204faa0243b4d0c0234a5f5f83a2549ecb5b7/pkgs/stdenv/generic/setup.sh#L677-L685
+      if [ "$NIX_SSL_CERT_FILE" == "/no-cert-file.crt" ]; then
```

Comment:
hmm.. why is this necessary if we're already "whitelisting" the env vars in `shell.go`?

Reply:
OpenSSL interprets set vs. unset differently:

- Unset = look in default places
- Set = always look in this file

If the user doesn't have the variables set in their normal shell, (which most don't) then the system default path is used. When `nix-shell --pure` sets them to an invalid path, it (intentionally) breaks the default.

Reply:
ugh, I see...